### PR TITLE
Trivial changes to arch-chroot and genfstab

### PR DESCRIPTION
**Pull request by alfredfo**
@peeweep @thesamesam

### CODE REVIEW

File: sys-apps/arch-chroot/metadata.xml [hunk 1]
```diff
@@ -5,6 +5,10 @@
       <email>cat@catcream.org</email>
       <name>Alfred Persson</name>
     </maintainer>
+	<maintainer type="person" proxied="yes">
```
Comment (thesamesam) [inline sys-apps/arch-chroot/metadata.xml:8]: Fix indentation

Comment (alfredfo) [reply sys-apps/arch-chroot/metadata.xml]: yeah saw it as I pushed and then I lost internet connection

Comment (alfredfo) [reply sys-apps/arch-chroot/metadata.xml]: Prayed you didn't see before I got it back

Comment (thesamesam) [reply sys-apps/arch-chroot/metadata.xml]: 🤣

**Pull request closed without merging**